### DISCUSSION TITLE
Qwen2 is already open source. How can I access it on Dify?

### DISCUSSION THREAD

**Discussion by lai-serena** [langgenius/dify#5040] [Help]
Qwen2 is already open source. How can I access it on Dify?Qwen2-72b install is not on this list
![pic](https://github.com/langgenius/dify/assets/44218061/aed9e1fe-e900-4f92-b1ae-942f9c46e817)

**Comment by du-bi-n** [comment]
我也想问这个问题

**Comment by Yash-1511** [comment]
You can use ollama to run qwen2-72b to running locally. 

[Qwen2 72b using ollama](https://ollama.com/library/qwen2)

#### How to configure ollama in dify
- https://docs.dify.ai/guides/model-configuration/ollama#download-and-launch-ollama

**Comment by Linij** [comment]
我也在等在线 api 模型的接入。

**Comment by berton1982** [comment]
同问
